app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -87,6 +91,11 @@ const StatusContent = React.createClass({
     this.setState({ hidden: !this.state.hidden });
   },
 
+  handleCollapsedClick (e) {
+    e.preventDefault();
+    this.setState({ collapsed: !this.state.collapsed });
```

Comment:
Is this setting `collapsed` to !null? Does that result in false? A little unsure of the value flow here.